src/App.tsx

```diff
@@ -1,223 +1,186 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React, {
+  useContext, useEffect, useMemo, useRef, useState,
+} from 'react';
+import classnames from 'classnames';
+import { createTodo, getTodos, deleteTodo } from './api/todos';
 import { AuthContext } from './components/Auth/AuthContext';
+import { Todo } from './types/Todo';
+import { Filters } from './types/Filters';
+import { Header } from './components/Auth/Header';
+import { TodoList } from './components/Auth/TodoList';
+import { FooterTodo } from './components/Auth/FooterTodo';
 
 export const App: React.FC = () => {
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   const user = useContext(AuthContext);
   const newTodoField = useRef<HTMLInputElement>(null);
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [value, setValue] = useState('');
+
+  const [currentFilter, setCurrentFilter] = useState(Filters.All);
+  const [isError, setisError] = useState(false);
+  const [isDelete, setisDelete] = useState(false);
+  const [isHidden, setisHidden] = useState(false);
+  const [isEmpty, setisEmpty] = useState(false);
+  const [isAdding, setisAdding] = useState(false);
+  const completedTodo = todos.filter((todo) => todo.completed);
+  const clearCompleted = () => {
+    completedTodo.filter((todo) => deleteTodo(todo.id)
+      .then()
+      .catch(() => setisDelete(true)));
+  };
+
+  const removTodo = (todoId: unknown) => setTodos(todos
```

Comment:
```suggestion
  const updateTodos = (todoId: number) => setTodos(todos
```
it's a little bit confusing to have `removTodo` and `removeTodo`, you can change to updateTodos,  for example, or use `setTodos` right in `.then`
also, I can suppose that that `todoId` have number type

Reply:
Yes, but the todoId was written like unkown type in fetchClient file, so I can't fix it